crop.py

```diff
@@ -1,6 +1,5 @@
-#!/usr/bin/python
-# Copyright (c) 2013 SUSE Linux Products GmbH
-# Author: Ludwig Nussel
```

Comment:
we don't track authors in other files. We have git for that. I don't mind keeping the author.